pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2565,10 +2612,73 @@ func (c *Cloud) getTaggedSecurityGroups() (map[string]*ec2.SecurityGroup, error)
 	return m, nil
 }
 
+// Almost identical to updateInstanceSecurityGroupsForLoadBalancer, but it only adds ssg rules to
+// instances' inbound rules.
+func (s *AWSCloud) updateInstanceSharedSecurityGroups(ssgID string, allInstances []*ec2.Instance) error {
+	if s.cfg.Global.DisableSecurityGroupIngress || s.cfg.Global.EnableSharedSecurityGroupIngress == false {
```

Comment:
Does it not suffice to say `if !s.cfg.Global.EnableSharedSecurityGroupIngress`?  They can't both be true I thought.

(I also find `a || b == false` to be ambiguous, and would prefer `a || !b` - presuming that is how the precedence works)